spec/lib/data_update_scripts/migrate_data_to_work_field_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210630041322_migrate_data_to_work_field.rb",
+)
+
+describe DataUpdateScripts::MigrateDataToWorkField, sidekiq: :inline do
+  let!(:user) { create(:user) }
+  let(:profile) { user.profile }
+
+  before do
+    ProfileField.find_or_create_by!(label: "Work", input_type: :text_field)
```

Comment:
This is necessary because the migrate to data work field doesn't/cannot assume the AddWorkProfileField had already run before we test here, but the sequential ordering by timestamp/filename ensures that these were ordered in production/development.

Took me a minute to fully verify that.